### ISSUE TITLE
"Consent to Civility" is a dead link

### ISSUE THREAD

**Issue by vmbrasseur** [builderscon/builderscon#25]
In [this section](https://github.com/builderscon/builderscon#index).

The link points to a file which doesn't exist. I suspect it's supposed to go to the Code of Conduct file but the file names are different enough that I can't be sure what's intended here.

If this is meant to go to the Code of Conduct file, +1 this in a comment and I'll send a PR to fix it.

**Comment by lestrrat** [comment]
```
If this is meant to go to the Code of Conduct file, +1 this in a comment and I'll send a PR to fix it.
```

Yes, but I'll do that now. Thanks for the offer :D

**Comment by lestrrat** [comment]
fixed in #26. Thanks!
